src/Sylius/Behat/Page/External/PaypalExpressCheckoutPage.php

```diff
@@ -31,6 +31,14 @@ public function __construct(Session $session, array $parameters, RepositoryInter
         $this->securityTokenRepository = $securityTokenRepository;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function authorize()
+    {
+        $this->getDriver()->visit($this->findAuthorizeToken()->getTargetUrl() . '?token=EC-2d9EV13959UR209410U&PayerID=UX8WBNYWGBVMG');
```

Comment:
Where these values came from? Is it just something random?

Reply:
They are from further down in the pay and cancel methods, just reused them